components/input/input.tsx

```diff
@@ -137,6 +139,10 @@ const Input = React.forwardRef<HTMLInputElement, InputProps>(
       if (disabled) return
       onIconClick && onIconClick(e)
     }
+    const iconRightClickHandler = (e: React.MouseEvent<HTMLDivElement>) => {
+      if (disabled) return
+      onIconRightClick && onIconRightClick(e)
+    }
     const iconProps = useMemo(
```

Comment:
or need to delete iconProps？

Reply:
The iconProps is the old object for the left icon.